src/Shared/Recruit.Vacancies.Client/Application/CommandHandlers/CreateApplicationReviewCommandHandler.cs

```diff
@@ -49,10 +58,10 @@ public async Task Handle(CreateApplicationReviewCommand message, CancellationTok
 
             await _applicationReviewRepository.CreateAsync(review);
 
-            await _messaging.PublishEvent(new ApplicationReviewCreatedEvent
+            await _messaging.PublishEvent(new ApplicationReviewUpdatedEvent
```

Comment:
This event feels a little strange on a create.

Reply:
Agree. I wanted to reuse the event rather than create another one. Maybe `ApplicationReviewChangedEvent` ? @leewadhams